packages/ui/primitives/theme-switcher.tsx

```diff
@@ -18,7 +18,7 @@ export const ThemeSwitcher = () => {
       >
         {isMounted && theme === THEMES_TYPE.LIGHT && (
           <motion.div
-            className="bg-background absolute inset-0 rounded-full mix-blend-exclusion"
+            className="bg-background absolute inset-0 rounded-full mix-blend-color-burn"
```

Comment:
Instead of changing the mix-blend for just one icon, the optimal solution would be to play around with `z-index`, we can remove mix-blend-exculsion from all the `divs` and give all the SVG a higher `z-index` than the `div` which is circling the SVG in the active state. That would fix it for all the icons.

Reply:
That makes sense, I will send an update for that

Reply:
I wouldn't do that, the mix blend mode hack is a trick for framer motion animations. See: https://www.youtube.com/watch?v=kep_Iaxuzy0